File: src/Example3Java/EX05.java
package Example3Java;

import java.util.Scanner;

public class EX05 {
    public static void main(String[] args) {
        System.out.print("Enter Text : ");
        String str = new Scanner(System.in).nextLine();
        String[] str1 = str.trim().split(" ");
        System.out.println("Text1 : " + str1[0] + " , Text2 : " + str1[str1.length - 1]);
    }
}

File: src/Example3Kotlin/EX05.kt
import java.util.*

object EX05 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter Text : ")
        val str = Scanner(System.`in`).nextLine()
        val str1 = str.trim().split(" ")
        println("Text1 : ${str1[0]}, Text2 : ${str1[str1.size - 1]}")
    }
}
